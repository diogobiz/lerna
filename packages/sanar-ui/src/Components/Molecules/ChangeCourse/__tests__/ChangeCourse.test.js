import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESChangeCourse from '../'

it('renders correctly', () => {
    const component = (
        <ESChangeCourse
            title='Trilha Sanar Enfermagem'
            date='23/05/2020'
            percent={45}
            coverPicture='http://sites.psu.edu/huangnutr360/files/2017/04/lesson-0-1ta118a.png'
            icon='https://public-v2links.adobecc.com/708e2f04-215d-454f-6692-c1d3b53d580f/component?params=component_id:2eef4ded-78c2-4ef5-abe4-08b602aad71c&params=version:0&token=1558008652_da39a3ee_7a22c22a02018c4fd70f9f9f69150074add489ff&api_key=CometServer1'
        />
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
