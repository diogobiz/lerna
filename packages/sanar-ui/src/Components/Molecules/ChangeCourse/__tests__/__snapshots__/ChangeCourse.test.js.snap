// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="es-change-course"
  onClick={[Function]}
  style={
    Object {
      "backgroundImage": "url(http://sites.psu.edu/huangnutr360/files/2017/04/lesson-0-1ta118a.png)",
      "cursor": "pointer",
    }
  }
>
  <div
    className="es-change-course__content"
  >
    <div
      className="d-flex align-items-center"
    >
      <div
        className="ant-skeleton es-skeleton es-change-course__content--skeleton es-skeleton--dark ant-skeleton-with-avatar ant-skeleton-active"
      >
        <div
          className="ant-skeleton-header"
        >
          <span
            className="ant-skeleton-avatar ant-skeleton-avatar-circle"
            style={
              Object {
                "height": 30,
                "lineHeight": "30px",
                "width": 30,
              }
            }
          />
        </div>
      </div>
      <img
        className="es-change-course__content--image"
        onLoad={[Function]}
        src="https://public-v2links.adobecc.com/708e2f04-215d-454f-6692-c1d3b53d580f/component?params=component_id:2eef4ded-78c2-4ef5-abe4-08b602aad71c&params=version:0&token=1558008652_da39a3ee_7a22c22a02018c4fd70f9f9f69150074add489ff&api_key=CometServer1"
        style={
          Object {
            "display": "none",
          }
        }
        width={30}
      />
      <div
        className="es-change-course__content--title"
      >
        <div
          className="es-typography text-white-10 es-typography--subtitle1 es-typography--strong es-typography--ellipsis"
          data-testid="es-typography"
        >
          Trilha Sanar Enfermagem
        </div>
        <div
          className="es-typography text-white-7 es-typography--caption"
          data-testid="es-typography"
        >
          Termina em: 23/05/2020
        </div>
      </div>
    </div>
  </div>
  <div
    className="es-change-course__progress"
  >
    <div
      className="es-progress es-progress--warning"
    >
      <div
        className="ant-progress ant-progress-line ant-progress-status-normal ant-progress-show-info ant-progress-small"
      >
        <div>
          <div
            className="ant-progress-outer"
          >
            <div
              className="ant-progress-inner"
            >
              <div
                className="ant-progress-bg"
                style={
                  Object {
                    "background": undefined,
                    "borderRadius": "",
                    "height": 6,
                    "width": "45%",
                  }
                }
              />
            </div>
          </div>
          <span
            className="ant-progress-text"
            title="45%"
          >
            45%
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
